src/Generation/FieldDetails.php

```diff
@@ -236,6 +243,15 @@ private function determineIsRequired(DescriptorProto $containingMessage, FieldDe
         return $isRequired;
     }
 
+    private function isFieldInfoUuid(FieldDescriptorProto $field)
+    {
+        $fieldInfo = ProtoHelpers::fieldInfo($field)->firstOrNull();
+        if ($fieldInfo) {
+            return $fieldInfo->getFormat() === 1;
```

Comment:
Please use the enum in the comparison rather than the magic number, I think its `Google\Api\FieldInfo\Format::UUID4`.

Reply:
Yeah this looks better. Updated it.